specification/ml/put_trained_model/MlPutTrainedModelRequest.ts

```diff
@@ -102,5 +103,11 @@ export interface Request extends RequestBase {
      * An array of tags to organize the model.
      */
     tags?: string[]
+    /**
+     * Optional prefix strings applied at inference
+     * @availability stack since=8.12.0
+     * @availability serverless since=8.12.0
```

Comment:
```suggestion
     * @availability serverless
```

There are no versions in serverless, and everybody runs the latest version, so I don't think this line should specify a version.